### DISCUSSION TITLE
Feature Request: Data Source Firebird

### DISCUSSION THREAD

**Discussion by Mueke666** [grafana/grafana#38825] [Data Sources]
Hello,

we need a new Data Source From a Firebird Database but there is no Plugin.
Can you make some Plugin for this Database please?

Greetings from Germay

**Comment by motogo** [comment]
Ja wäre super wenn es ein firebird Plugin für grafana gäbe

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82978. Feel free to continue the discussion around this there. Thank you!
